.changeset/real-spies-pretend.md

```diff
@@ -2,27 +2,29 @@
 '@astrojs/tailwind': major
 ---
 
-Let tailwind postcss plugin load its config file itself. This changes the `tailwind.config.js` loading behaviour where Tailwind would load the config file from `process.cwd()` instead of the project `root`. You can configure the integration's `config.path` option to load from a specific path instead.
+Let tailwind postcss plugin load its config file itself. This changes the Tailwind config loading behaviour where it is loaded from `process.cwd()` instead of the project `root`.
```

Comment:
Only because Matthew now needs better edited changesets... 😆 ...

"tailwind postcss plugin" is either missing capitalization of these proper nouns, OR if that's the name of the plugin, I'd use it directly in code backticks e.g. `tailwind-postcss` (< - I prefer this option if true, a little, not strongly.)

```suggestion
Let Tailwind PostCSS plugin load its config file itself. This changes the Tailwind config loading behaviour where it is loaded from `process.cwd()` instead of the project `root`.
```

Reply:
Yeah a lot of this is written in a semi-informal way 😄 It is the `tailwindcss` PostCSS plugin, so we could use that instead:
```suggestion
Let the `tailwindcss` PostCSS plugin load its config file itself. This changes the Tailwind config loading behaviour where it is loaded from `process.cwd()` instead of the project `root`.
```